ds-jupyter-notebook/Jenkinsfile

```diff
@@ -16,7 +16,7 @@ odsQuickstarterPipeline(
       --component ${context.componentId} \
       --nexus-url ${context.nexusUrl} \
       --nexus-username ${context.nexusUsername} \
-      --nexus-password ${context.nexusPassword} \
+      --nexus-password b3BlbnNoaWZ0 \
```

Comment:
this hack still requires fixing ...

Reply:
@georgfedermann  - what's the problem here? the password must be set - I guess it#s not base64 encoded, right?